### PR TITLE
Support Terminus plugins in Github and Gitlab

### PR DESCRIPTION
**Pull request by rabbitlair**
This PR just makes the [Terminus Secrets plugin](https://github.com/pantheon-systems/terminus-secrets-plugin) available by default when installing Drainpipe

### CODE REVIEW

File: scaffold/github/actions/pantheon/setup-terminus/action.yml [hunk 1]
```diff
@@ -13,5 +13,6 @@ runs:
         curl -L https://github.com/pantheon-systems/terminus/releases/download/3.0.7/terminus.phar --output ~/terminus/terminus
         chmod +x ~/terminus/terminus
         ln -s ~/terminus/terminus /usr/local/bin/terminus
+        terminus self:plugin:install pantheon-systems/terminus-secrets-plugin
```
Comment (deviantintegral) [inline scaffold/github/actions/pantheon/setup-terminus/action.yml:16]: Should these specify a specific version to install?

**Review by deviantintegral — COMMENTED** [review]
My only other unrelated thought is that it feels odd to have two different copies of what looks to be the same code for downloading and installing terminus.

**Comment by deviantintegral** [comment]
I was more just thinking that perhaps both CI files call into a shared task or shell script. Regardless, it's not something needed for this PR.

Leaving for @justafish to double check!

**Review by justafish — APPROVED** [review]
fantastic! :tada:

**Pull request merged**